chart/epinio/values.yaml

```diff
@@ -174,5 +179,7 @@ global:
   # The URL of the container registry from where to pull container images for the various
   # created Pods. Don't confuse this registry with the "Epinio registry" which is the one
   # where Epinio stores the application images.
+  tlsIssuerMail: "epinio@suse.com"
+  # Email address to receive the certificate notification emails send by the `letsencrypt-production` issuer.
```

Comment:
I guess that the comment should be right above, keeping the old one to the `cattle`.

I'm also not sure if it should be `tlsIssuerEmail`, instead of `tlsIssuerMail`.

Reply:
... Rereading ... Right, I somehow thought comments were after the thing
Yes, email is likely better.